docs.html

```diff
@@ -12,17 +12,18 @@
   <link rel="stylesheet" href="./Components/Badge Component/badge.css">
   <link rel="stylesheet" href="./Components/Button Component/button.css">
   <link rel="stylesheet" href="./Components/Input Component/input.css">
-   <link rel="stylesheet" href="./Components/Text Utility Component/text-utility.css">
+  <link rel="stylesheet" href="./Components/Text Utility Component/text-utility.css">
   <link rel="stylesheet" href="./Components/List Component/list.css">
   <link rel="stylesheet" href="./Components/Navigation Component/navigation.css">
+  <link rel="stylesheet" href="./Components/Modal Component/modal.css">
   <link rel="stylesheet" href="docs.css">
```

Comment:
Here I think another good way can be to import these all stylesheets in one single file CSS file and then reference just that single CSS file here and subsequently in all the HTML files.

Reply:
thank you for the suggestion, did as you suggested